plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/ReindexResponse.java

```diff
@@ -0,0 +1,84 @@
+package org.elasticsearch.plugin.reindex;
+
+import org.elasticsearch.action.bulk.BulkItemResponse.Failure;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentBuilderString;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+
+import static org.elasticsearch.plugin.reindex.BulkIndexByScrollResponse.Fields.CREATED;
+
+public class ReindexResponse extends BulkIndexByScrollResponse {
+    private long created;
+
+    public ReindexResponse() {
+    }
+
+    public ReindexResponse(long took, long created, long updated, int batches, long versionConflicts, long noops, List<Failure> failures) {
+        super(took, updated, batches, versionConflicts, noops, failures);
+        this.created = created;
+    }
+
+    public long created() {
+        return created;
+    }
+
+    @Override
+    public void writeTo(StreamOutput out) throws IOException {
+        // NOCOMMIT need a round trip test for this
+        super.writeTo(out);
+        out.writeVLong(created);
+    }
+
+    @Override
+    public void readFrom(StreamInput in) throws IOException {
+        super.readFrom(in);
+        created = in.readVLong();
+    }
+
+    static final class Fields {
+        static final XContentBuilderString CREATED = new XContentBuilderString("created");
```

Comment:
can we just make these `XContentBuilderString` static a `static final String` in the enclosing class? I know we do this everywhere but it's really just confusing and not helpful.